tests/sanity/tests/actions.spec.ts

```diff
@@ -10,19 +10,39 @@ test.describe('actions tests', () => {
     // Create user and workspace
     await (await page.goto(`${PlatformURI}/workbench/sanity-ws/sanity-ws`))?.finished()
   })
-  test('action-new-candidate', async ({ page }) => {
-    await page.click('[id="app-recruit\\:string\\:RecruitApplication"]')
 
-    await page.click('text=Talents')
-    await expect(page).toHaveURL(`${PlatformURI}/workbench/sanity-ws/recruit/talents`)
+  test('Add new candidate', async ({ page, authorizedMainPage, recruitPage }) => {
+    await test.step(`Open recruit application`, async () => {
+      await authorizedMainPage.leftSidebar.openRecruitApplication();
+    });
 
-    await page.click('td:has-text("Frontend Engineer")')
+    await test.step(`Open talents tab`, async () => {
+      await recruitPage.recruitsPanel.openTalentsTab();
+    });
 
-    await page.press('body', 'Meta+k')
-    await page.fill('[placeholder="type\\ to\\ filter\\.\\.\\."]', 'Talent')
-    expect(await page.locator('div.selectPopup :text("New Talent")').count()).toBe(1)
-    await page.click('div.selectPopup :text("New Talent")')
-    await page.click('button#card-close')
+    await test.step(`Check talents url`, async () => {
+      await expect(page).toHaveURL(`${PlatformURI}/workbench/sanity-ws/recruit/talents`)
```

Comment:
nit: I know that it was in the original test, but I don't see a purpose of checking URL in the test that verifies a candidate creation.

I see the following option:
1. Remove this check
2. If URLs are important for some reason, we can add a separate test specific to navigation
3. Or move this check into `openTalentsTab` so we check URL each time we navigated.

Reply:
I saved it because it was in original test. These kind of checks needs to be in other Assert classes, and if I were you I would use these checks. Maybe later I'll send example.